tests/test_subtractTask.py

```diff
@@ -715,6 +715,76 @@ def _compare_apCorrMaps(self, a, b):
             self.assertFloatsAlmostEqual(
                 value.getCoefficients(), value2.getCoefficients(), rtol=0.0)
 
+    def test_fake_mask_plane_propagation(self):
+        """Test that we have the mask planes related to fakes in diffim images.
+        This is testing method called updateMasks
+        """
+        xSize = 200
+        ySize = 200
+        science, sources = makeTestImage(psfSize=2.4, xSize=xSize, ySize=ySize)
+        science_fake_img, science_fake_sources = makeTestImage(
+            psfSize=2.4, xSize=xSize, ySize=ySize, seed=7, nSrc=2, noiseLevel=0.25, fluxRange=1
+        )
+        template, _ = makeTestImage(psfSize=2.4, xSize=xSize, ySize=ySize, doApplyCalibration=True)
+        tmplt_fake_img, tmplt_fake_sources = makeTestImage(
+            psfSize=2.4, xSize=xSize, ySize=ySize, seed=9, nSrc=2, noiseLevel=0.25, fluxRange=1
+        )
+        # created fakes and added them to the images
+        science.image.array += science_fake_img.image.array
+        template.image.array += tmplt_fake_img.image.array
+
+        # TODO: update to INJECTED names when source injection gets refactored
+        # adding mask planes to both science and template images
+        science.mask.addMaskPlane("FAKE")
+        template.mask.addMaskPlane("FAKE")
+
+        science_mask_planes = science.mask.getMaskPlaneDict()
+        template_mask_planes = template.mask.getMaskPlaneDict()
+
+        x0 = science.mask.getX0()
+        y0 = science.mask.getY0()
+        for a_science_source in science_fake_sources:
+            # 3 x 3 masking of the source locations is fine
+            for dy in (-1, 0, 1):
+                science.mask.setMaskPlaneValues(
+                    science_mask_planes['FAKE'],
+                    int(a_science_source.getX() - x0)-1,
+                    int(a_science_source.getX() - x0)+1,
+                    int(a_science_source.getY() - y0)+dy
+                )
```

Comment:
This is fine, but I have an alternate suggestion:
```
bbox = lsst.geom.Box2I(lsst.geom.Point2I(a_science_source.getX(), a_science_source.getY()), lsst.geom.Extent2I(3, 3))
science[bbox].mask.array |= science_fake_mask
```
using `science_fake_mask` from my above suggestion.